lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -809,7 +809,6 @@
     from cryptography import x509
     from cryptography.hazmat.backends import default_backend
     from cryptography.hazmat.primitives.serialization import Encoding
-    from cryptography.hazmat.primitives.hashes import SHA1
```

Comment:
This was added for Entrust, right?

Reply:
It was, but it wasn't used there. (I think it was used in an older version of that PR, but the code using it was removed during review.)